org.hlwd.biblemultithelife.yml

```diff
@@ -0,0 +1,48 @@
+app-id: org.hlwd.biblemultithelife
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: biblemultithelife
+separate-locales: false
+finish-args:
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-run/xdg-desktop-portal
```

Comment:
what is this for? 
```suggestion
```

Reply:
Το give access to open uri, but OK, I remove everything and retest locally.

Reply:
where in the documentation does it say to do that?

Reply:
chatgpt, but I removed it :)